### PR TITLE
Add Github desktop alternative to readme

### PR DESCRIPTION
**Pull request by TrystonMinsquero**
Simply change the wording also to include GitHub Desktop alternatives

Added GitHub Desktop alternative for people

## Please complete this quality control checklist. 
- [x] I am a student at The University of Texas at Dallas.
- [x] My changes generate no new errors or warnings.
- [x] I have performed a self-review of my changes.
- [x] I was able to build the project without issues.

### CODE REVIEW

**Review by Phu-D-Le — DISMISSED** [review]
lgtm

File: README.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ These guidelines will help you get started with the project. However, they can b
 ### Prerequisites
 
 - [Unity Hub](https://unity.com/) with [Unity 2021.3.3f1](https://unity3d.com/get-unity/download/archive) installed
-- [GitKraken Client](https://www.gitkraken.com/) or another Git client
+- [GitKraken Client](https://www.gitkraken.com/), [Github Desktop](https://desktop.github.com/), or another Git client
```
Comment (EMERALD0874) [inline README.md]:
```suggestion
[GitKraken Client](https://www.gitkraken.com/), [GitHub Desktop](https://desktop.github.com/), or another Git client
```
Should be "GitHub Desktop".

File: README.md [hunk 2]
```diff
@@ -35,7 +35,7 @@ Once you have cloned the repository, you can open the project in Unity. You can
 
 ### Contributing
 
-To start, create a branch in GitKraken by clicking the "Branch" button. Then, enter a name for your branch and click "Create Branch". You can now make changes to the project. Once you are done, you can commit your changes by clicking the "Commit" button. Then, enter a message for your commit and click "Commit to Branch". You can now push your changes to the repository by clicking the "Push" button.
+To start, create a branch in GitKraken or Github Desktop by clicking the "Branch" button. Then, enter a name for your branch and click "Create Branch". You can now make changes to the project. Once you are done, you can commit your changes by clicking the "Commit" button. Then, enter a message for your commit and click "Commit to Branch". You can now push your changes to the repository by clicking the "Push" button.
```
Comment (EMERALD0874) [inline README.md]:
```suggestion
To start, create a branch in GitKraken by clicking the "Branch" button. Then, enter a name for your branch and click "Create Branch". You can now make changes to the project. Once you are done, you can commit your changes by clicking the "Commit" button. Then, enter a message for your commit and click "Commit to Branch". You can now push your changes to the repository by clicking the "Push" button.
```
GitHub Desktop uses a different setup for creating and managing branches, so these steps are invalid.

**Review by EMERALD0874 — CHANGES_REQUESTED** [review]
I left some changes that I think should be made before merging this into main.

File: README.md (hunk 2 above)
Comment (TrystonMinsquero) [reply README.md]: It uses the same thing, I checked. Or it's similar enough to where it's not confusing

Comment (EMERALD0874) [reply README.md]:
```suggestion
To start, create a branch in GitKraken by clicking the "Branch" button. Then, enter a name for your branch and click "Create Branch". You can now make changes to the project. Once you are done, you can commit your changes by clicking the "Commit" button. Then, enter a message for your commit and click "Commit to Branch". You can now push your changes to the repository by clicking the "Push" button. Similar steps can be followed using GitHub Desktop.
```
I'd rather make sure our documentation is as specific as possible. Freshmen and people unfamiliar with version control may be working on this project. I'm okay with just saying that similar steps exist for GitHub Desktop if that works with you.

**Review by EMERALD0874 — CHANGES_REQUESTED** [review]
Changed my suggestion for the branch instructions.

**Review by EMERALD0874 — APPROVED** [review]
LGTM

**Pull request merged**